### DISCUSSION TITLE
Show Replied Message in Bubble Chat

### DISCUSSION THREAD

**Discussion by fathorrahmans** [flyerhq/flutter_chat_ui#534] [Q&A] [answered]
how to display a chat bubble which also contains the chat being replied to?

**Comment by demchenkoalex** [comment]
replies is in a scope of `v2` version which is under construction 🏗️

**Comment by demchenkoalex** [comment, marked as answer]
for v2 pointing to this https://github.com/flyerhq/flutter_chat_ui/discussions/97
